io.c

```diff
@@ -4317,11 +4318,11 @@ rb_io_set_lineno(VALUE io, VALUE lineno)
 
 /*
  *  call-seq:
- *    readline(sep = $/, **line_opts)   -> string
- *    readline(limit, **line_opts)      -> string
- *    readline(sep, limit, **line_opts) -> string
+ *    readline(sep = $/, chomp: false)   -> string
+ *    readline(limit, chomp: false)      -> string
+ *    readline(sep, limit, chomp: false) -> string
  *
- *  Reads a line as with IO#gets, but raises EOFError if already at end-of-file.
+ *  Reads a line as with IO#gets, but raises EOFError if already at end-of-stream.
  *
```

Comment:
Can you add a short paragraph on what `chomp` does? Something like:

```suggestion
 *  Optional keyword argument +chomp+ specifies whether line separators
 *  are to be omitted.
```

Reply:
Done.